x-pack/legacy/plugins/infra/public/components/source_configuration/log_columns_configuration_form_state.tsx

```diff
@@ -106,6 +106,18 @@ export const useLogColumnsConfigurationFormState = ({
     [formState.logColumns]
   );
 
+  const reorderLogColumns = useCallback(
```

Comment:
Would something like `moveLogColumn` be more descriptive of the specific reordering that takes place?